src/containers/table/index.js

```diff
@@ -212,7 +215,14 @@ class TableComponent extends Component {
                                                     textAlign: 'right',
                                                     margin: '0 auto',
                                                   }}>
-                                                  {paginationProps.startIndex + index1 + 1}
+                                                  {props.enableIcon ? (
+                                                    <>
+                                                      {paginationProps.startIndex + index1 + 1}
+                                                      {row[isIcon] ? props.onTrue : props.onFalse}
```

Comment:
This works only in case of a boolean, as discussed with you move this to tables where u return which icon has to be returned depending on the condition to make it a generic one